RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chrome.java

```diff
@@ -223,8 +241,7 @@ private void getHistory() {
                         RecentActivityExtracterModuleFactory.getModuleName(),
                         ((result.get("title").toString() != null) ? result.get("title").toString() : ""))); //NON-NLS
                 bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_PROG_NAME,
-                        RecentActivityExtracterModuleFactory.getModuleName(),
-                        NbBundle.getMessage(this.getClass(), "Chrome.moduleName")));
+                        RecentActivityExtracterModuleFactory.getModuleName(), browser));
```

Comment:
If these browser names are going to be displayed to the user, I think they should be put into a Bundle so they can be translated.

Reply:
Done